### DISCUSSION TITLE
IntelliJ doesn't like intents.message_content

### DISCUSSION THREAD

**Discussion by Keiishu** [DisnakeDev/disnake#447] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/28761304/161878997-f019d61a-e303-4332-8138-962621a319f9.png)

It still works tho. Am I doing something wrong?

I'm using the repo version (installed from repo clone).

**Comment by onerandomusername** [comment, marked as answer]
Sorry for the late response, I just saw this. You may need to restart your editor after installing or updating dependencies to have the language server recache your dependencies.

**Reply by Keiishu** [reply]
Somehow that didn't worked for me at the time, but it's fine now. Thanks!
